src/proto/semantifly.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+package semantifly;
+
+import "index.proto";
+
+option go_package = "accretional.com/semantifly/proto";
+
+service Semantifly {
+  rpc Add(AddRequest) returns (AddResponse) {}
+  rpc Delete(DeleteRequest) returns (DeleteResponse) {}
+  rpc Get(GetRequest) returns (GetResponse) {}
+  rpc Update(UpdateRequest) returns (UpdateResponse) {}
+  rpc LexicalSearch(LexicalSearchRequest) returns (LexicalSearchResponse) {}
+}
+
+message AddRequest {
+  repeated ContentMetadata files_data = 1;
```

Comment:
It seems weird to have this be repeated but none of the other CRUD rpcs take repeated args, but let's not address this yet unless it's easy

Reply:
fixed